
Of course the shared library needs PIC - imported from ROCK Linux.

  - Rene Rebe <rene?@exactcode.de>

diff -ruN bzip2-1.0.2-orig/Makefile bzip2-1.0.2/Makefile
--- bzip2-1.0.2-orig/Makefile	2004-11-07 04:19:14.950033312 +0100
+++ bzip2-1.0.2/Makefile	2004-11-07 04:20:26.998080344 +0100
@@ -9,7 +9,7 @@
 
 # Suitably paranoid flags to avoid bugs in gcc-2.7
 BIGFILES=-D_FILE_OFFSET_BITS=64
-CFLAGS=-Wall -Winline -O2 -fomit-frame-pointer -fno-strength-reduce $(BIGFILES)
+CFLAGS=-Wall -Winline -fPIC -O2 -fomit-frame-pointer -fno-strength-reduce $(BIGFILES)
 
 # Where you want it installed when you do 'make install'
 PREFIX=/usr
